config.example/kube-prometheus.yml

```diff
@@ -0,0 +1,70 @@
+prometheus:
+  ingress:
+    enabled: true
+    hosts:
+    - prometheus.local
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+      nginx.ingress.kubernetes.io/ssl-redirect: "false"
+      nginx.ingress.kubernetes.io/rewrite-target: /
+    # Support HTTPS (required)
+    tls:
+    - hosts:
+      -  prometheus.local
+  nodeSelector:
+    node-role.kubernetes.io/master: "true"
+  service:
+    type: NodePort
+    nodePort: 30500
+  serviceMonitors:
+  - name: dcgm-exporter
+    selector:
+      matchLabels:
+        app: dcgm-exporter
+    endpoints:
+    - port: scrape
+      interval: 1s
+      honorLabels: true       
+  serviceMonitorsSelector:
+    matchLabels:
+      prometheus: "kube-prometheus"
+
+alertmanager:
+  ingress:
+    enabled: true
+    hosts:
+    - alertmanager.local
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+      nginx.ingress.kubernetes.io/ssl-redirect: "false"
+      nginx.ingress.kubernetes.io/rewrite-target: /
+    # Support HTTPS (required)
+    tls:
+    - hosts:
+      -  alertmanager.local
+  nodeSelector:
+    node-role.kubernetes.io/master: "true"
+  service:
+    type: NodePort
```

Comment:
Not sure I'm in love with NodePorts for the AlertManager and general Prometheus instance.  

Perhaps we add Istio and create proper ingress routes.